webpack.config.ts

```diff
@@ -523,8 +523,6 @@ if (
 
   appConfig.devServer = {
     headers: {
-      'Access-Control-Allow-Origin': '*',
-      'Access-Control-Allow-Credentials': 'true',
```

Comment:
These headers were overwriting headers from the django server. This combination is also invalid as you can't send credentials when allow-origin is `*`.